Error when using this package in a TS environment

`import collapse from 'svelte-collapse';` prompts the ts error 2307 `Cannot find module 'svelte-collapse' or its corresponding type declarations`.

I think you're missing something in your package.json, like "main" or "exports", and having types would be awesome. I don't know much about publishing packages, even less so about publishing svelte action packages, so I can't make you a PR for this.

As a workaround for myself I just copied your collapse.js file to my own codebase and made it to a TS file. If you ever want to switch to TS for this lib, or if someone else has this problem, the file is here, adapted from version 0.1.1: https://gist.github.com/Apollinaire/b27c17f96517428a60a003a10986263f

--- a/package.json
+++ b/package.json
@@ -1,8 +1,9 @@
 {
   "name": "svelte-collapse",
-  "version": "0.1.1",
+  "version": "0.1.2",
   "license": "MIT",
   "svelte": "src/index.js",
+  "types": "src/index.d.ts",
   "scripts": {
     "build": "rollup -c",
     "dev": "rollup -c -w",

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -1,0 +1,11 @@
+declare module "svelte-collapse" {
+  import type { ActionReturn } from "svelte/types/runtime/action";
+
+  interface CollapseParams {
+	open: boolean;
+	duration: number;
+	easing: string;
+  }
+
+  export default function collapse(node: HTMLElement, params: CollapseParams): ActionReturn;
+}

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -7,5 +7,5 @@ declare module "svelte-collapse" {
 	easing: string;
   }
 
-  export default function collapse(node: HTMLElement, params: CollapseParams): ActionReturn;
+  export default function collapse(node: HTMLElement, params: Partial<CollapseParams>): ActionReturn;
 }

